### PR TITLE
Add Chat screen

### PR DESCRIPTION
**Pull request by aspasionov**
image
image

### CODE REVIEW

File: src/components/Layout/index.jsx [hunk 1]
```diff
@@ -24,19 +24,19 @@ const Layout = ({ children }) => {
...
-    },
     {
       id: 5,
       text: 'Chat',
       Icon: <ChatIcon/>,
       path: 'chat',
       protected: true
+    },
+    {
+      id: 6,
```
Comment (SerhiienkoRuslan) [inline src/components/Layout/index.jsx:35]: Use words for id
It`s confused when your ids looks like - 3,5,6

File: src/components/Preloader/index.jsx [hunk 2]
```diff
@@ -1,15 +1,9 @@
...
-    <Container>
-      <Circle />
-      <Circle />
-      <Circle />
-      <Circle />
-    </Container>
-  </Box>
+const Preloader = () => (
+  <div className='absolute left-0 top-0 right-0 bottom-0 flex bg-white/[.5] z-30'>
+    <div className="lds-hourglass m-auto"/>
```
Comment (SerhiienkoRuslan) [inline src/components/Preloader/index.jsx:5]: Nice!

File: src/screens/Posts/components/Post/index.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import PropTypes from 'prop-types';
+
+const Post = ({ post }) => (
+    <div key={post?.id} className='ml-[2%] mb-[2%] w-[31%] p-4 bg-white rounded-md shadow-md text-black'>
+      <h3 className='font-bold mb-2 text-lg'>{post?.title}</h3>
+
+      <p className='text-base'>{post?.content}</p>
+
+      <div className=' flex mt-2 justify-between border-black border-2 rounded-md p-2'>
+        <div className='mb-1 font-medium text-xs w-[48%] truncate' title={post?.author?.username}>
```
Comment (SerhiienkoRuslan) [inline src/screens/Posts/components/Post/index.jsx:11]: Should redirect to user page

**Comment by nester7x** [comment]
image
try not to use eslint-disable-next-line
check all files and fix it

**Pull request merged**